modules/cluster-logging-loki-deploy.adoc

```diff
@@ -8,7 +8,7 @@ You can use the {product-title} web console to deploy the LokiStack.
 .Prerequisites
 
 * {logging-title-uc} Operator 5.5 and later
-* AWS S3 bucket for log storage
+* AWS S3 compatible bucket for log storage
```

Comment:
**NOTE:** GitHub will make a button for this comment that says something like "accept suggestion". Don't click it, it does weird stuff that's harder to get through than just editing normally :slightly_smiling_face: 

Need a hyphen for the compound adjective here:
```suggestion
* AWS S3-compatible bucket for log storage
```